src/App/ElementMap.h

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+/****************************************************************************
+ *   Copyright (c) 2018-2022 Zheng, Lei (realthunder)                       *
+ *   <realthunder.dev@gmail.com>                                            *
+ *   Copyright (c) 2023 FreeCAD Project Association                         *
+ *                                                                          *
+ *   This file is part of FreeCAD.                                          *
+ *                                                                          *
+ *   FreeCAD is free software: you can redistribute it and/or modify it     *
+ *   under the terms of the GNU Lesser General Public License as            *
+ *   published by the Free Software Foundation, either version 2.1 of the   *
+ *   License, or (at your option) any later version.                        *
+ *                                                                          *
+ *   FreeCAD is distributed in the hope that it will be useful, but         *
+ *   WITHOUT ANY WARRANTY; without even the implied warranty of             *
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU       *
+ *   Lesser General Public License for more details.                        *
+ *                                                                          *
+ *   You should have received a copy of the GNU Lesser General Public       *
+ *   License along with FreeCAD. If not, see                                *
+ *   <https://www.gnu.org/licenses/>.                                       *
+ *                                                                          *
+ ***************************************************************************/
+
+#ifndef DATA_ELEMENTMAP_H
+#define DATA_ELEMENTMAP_H
+
+#include "FCGlobal.h"
+#include "IndexedName.h"
+
+namespace Data {
+
+const std::string POSTFIX_TAG = ";:H";
+const std::string POSTFIX_DEC_TAG = ";:T";
```

Comment:
Maybe call this POSTFIX_DECIMAL_TAG for more clarity? At least to me it's not immediately obvious what dec stands for, could just be me though. Might be worth adding a comment about this being there only for backwards compatibility.